### DISCUSSION TITLE
После завершения установки содержимое веб-страницы отображается не полностью

### DISCUSSION THREAD

**Discussion by snmin** [totumonline/totum-ru-issues-and-discussions#90] [Как это сделать?] [answered]
![image](https://user-images.githubusercontent.com/44972156/156887994-5eb8c628-4795-47a7-9c56-f7021dcddadd.png)
Здравствуйте, (я из Китая, русский язык переводится через программу-переводчик) После установки отображается веб-страница и содержимое неполное. Последующие операции выполнить невозможно. При нажатии на "Системные таблицы" отображается ошибка 404. Я установил его в соответствии с руководством, и ОС использует ubuntu20.04.
     Я столкнулся с этой проблемой несколько месяцев назад, пытаясь установить, но не нашел решения. Надеюсь получить вашу поддержку. Спасибо!

**Comment by snmin** [comment]
кто мне подскажет

**Reply by ttm-alexeypolunin** [reply]
Docker install will be done by the end of this week. I think it will solve most of the issuies at install. I will write in this discussion when you can try to install with docker.

**Comment by snmin** [comment]
little advice......

Thank you for your reply, I saw your latest docker project on github, and found in "docker-compose.yml" that you need to use port 25 of the server. In actual use, many servers or network service providers will close port 25 by default. It is recommended to use ports such as 465 (SMTPS), 995 (POP3S), and 993 (IMAPS) to provide mail sending and receiving services.
In addition, is it possible to set not to use nginx and apache. Running nginx reverse proxy directly through the server to publish the website? In practice, many server management panels have integrated nginx.
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
original Chinese ：（小小建议）

谢谢你的回复，我在github上看到了你们最新的docker项目，在"docker-compose.yml " 中发现需要使用到服务器25端口。在实际使用中很多服务器或者网络服务供应商都会默认关闭25端口，建议改用 465 （SMTPS）、995（POP3S） 、 993（IMAPS） 等端口来提供邮件的收发服务。
另外，是否可以设置不使用nginx 、apache 。运行直接通过服务器本地的nginx 反向代理来发布网站了？在实际运用中，很多服务器的管理面板已经集成了nginx服务。

---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Russian translation ：（маленький совет）

Спасибо за ваш ответ, я видел ваш последний проект docker на github и нашел в «docker-compose.yml», что вам нужно использовать порт 25 сервера. При фактическом использовании многие серверы или поставщики сетевых услуг по умолчанию закрывают порт 25. Рекомендуется использовать такие порты, как 465 (SMTPS), 995 (POP3S) и 993 (IMAPS), для предоставления услуг отправки и получения почты.
Кроме того, можно ли настроить не использовать nginx и apache. Запуск обратного прокси-сервера nginx напрямую через сервер для публикации веб-сайта? На практике в панели управления многих серверов встроен сервис nginx.




***@***.***
 
发件人： Alexey Polunin
发送时间： 2022-03-08 16:45
收件人： totumonline/totum-ru-issues-and-discussions
抄送： snmin; Author
主题： Re: [totumonline/totum-ru-issues-and-discussions] После завершения установки содержимое веб-страницы отображается не полностью (Discussion #90)
Docker install will be done by the end of this week. I think it will solve most of the issuies at install. I will write in this discussion when you can try to install with docker.

**Reply by ttm-alexeypolunin** [reply]
Thanks, i will think about 495 port, but now we use fork from namshi/docker-smtp and use they instruction for use.

totum-mit-docker intended for use on clear system without panels for use by people, who have minimal skills in terminal. That's why we use full set of nginx-fpm-postgres-cron-certbot inside docker image.

I think that today i will fix the last problem with Certbot and tomorrow will publish the instruction.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Теперь есть Docker-установка в 1 клик

Now we have 1-Click Docker install

https://github.com/totumonline/totum-mit-docker

**Reply by mihail-cornev** [reply]
> Теперь есть Docker-установка в 1 клик
> Now we have 1-Click Docker install
> https://github.com/totumonline/totum-mit-docker

Я пробую установить, а система даёт ошибку. В этот раз решил установить имея доменное имя.
![Screenshot_1](https://user-images.githubusercontent.com/113771444/205706465-a1dc730d-0b05-48db-805d-1b3ef06d4daa.jpg)

Это я туплю или с установочным файлом чё-то?

**Reply by ttm-alexeypolunin** [reply]
по этой картинке непонятно, что вы делаете и поставьте лучше не docker — не нужен он.

```
sudo curl -O https://raw.githubusercontent.com/totumonline/totum-mit/master/totum/moduls/install/autoinstall.sh && sudo bash autoinstall.sh
```

**Reply by mihail-cornev** [reply]
> по этой картинке непонятно, что вы делаете и поставьте лучше не docker — не нужен он.
> 
> ```
> sudo curl -O https://raw.githubusercontent.com/totumonline/totum-mit/master/totum/moduls/install/autoinstall.sh && sudo bash autoinstall.sh
> ```

я ставлю именно по этой ссылке, которую вы указали.
просто захожу через root и пишу sudo curl -O https://raw.githubusercontent.com/totumonline/totum-mit/master/totum/moduls/install/autoinstall.sh && sudo bash autoinstall.sh

upd:
скачал сторонний терминал ssh и получилось, правда не знаю это ли помогло.

**Reply by ttm-alexeypolunin** [reply]
я проверил, скрипты работают

**Reply by mihail-cornev** [reply]
> я проверил, скрипты работают

спасибо, всё получилось.

**Comment by snmin** [comment]
Hi, great to hear from you.

Can you provide translation documents that are convenient for localization? I have already tried to translate some of the files, but the results are mediocre. If a unified translation document can be provided, more people will like totum-mit.


I ran into a few issues while testing your docker container installation.

Hint DISABLE_IPV6 contains true, which is an invalid type, it should be a string, number or null.
***@***.***:/rom/run/totum/totum-mit-docker# docker-compose up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.exim.environment.DISABLE_IPV6 contains true, which is an invalid type, it should be a string, number, or a null

After entering the information, click "Create config and upload scheme" prompt:

A scheme exists - choose another one to install. #0 /var/www/totum-mit/totum/common/TotumInstall.php(299): totum\common\TotumInstall->checkSchemaExists() #1 /var/www/totum-mit/totum/common/TotumInstall.php(191): totum\common\TotumInstall->createSchema() #2 /var/www/totum-mit/totum/moduls/install/installController.php(43): totum\common\TotumInstall->install() #3 /var/www/totum-mit/totum/moduls/install/installController.php(65): totum\moduls\install\installController->actionMain() #4 /var/www/totum-mit/http/index.php(36): totum\moduls\install\installController->doIt() #5 {main}




***@***.***
 
发件人： Alexey Polunin
发送时间： 2022-03-10 16:32
收件人： totumonline/totum-ru-issues-and-discussions
抄送： snmin; Author
主题： Re: [totumonline/totum-ru-issues-and-discussions] После завершения установки содержимое веб-страницы отображается не полностью (Discussion #90)
Thanks, i will think about 495 port, but now we use fork from namshi/docker-smtp and use they instruction for use.
totum-mit-docker intended for use on clear system without panels for use by people, who have minimal skills in terminal. That's why we use full set of nginx-fpm-postgres-cron-certbot inside docker image.
I think that today i will fix the last problem with Certbot and tomorrow will publish the instruction.

**Comment by ttm-alexeypolunin** [comment]
I fixed «true as string» and check this command from this repo: https://github.com/totumonline/totum-mit-docker

```
sudo curl -O https://raw.githubusercontent.com/totumonline/totum-mit-docker/main/autoinstall.sh && sudo bash autoinstall.sh
```

Everything works correctly. But we not recommend now to install docker version, because you cant switch it to PRO. Use native autoinstall script from main repo: https://github.com/totumonline/totum-mit

Info about translation you can find here: https://github.com/totumonline/totum-mit/discussions/228
